### PR TITLE
Display menu on mobile view pull request

### PR DESCRIPTION
**Pull request by AleWaweru**
added javascript file that controls display of nav menu on mobile view.
added a .eslintrc.json file

### CODE REVIEW

File: style3.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  left: 0;
+  width: 100%;
+  flex-direction: column;
+  justify-content: center;
+  gap: 3rem;
+  align-items: center;
+  background: #5e5b5b;
+  transition: top 0.5s ease;
+  display: none;
+}
```
Comment (zieeco) [inline style3.css]: - [ ]  Hey, please, let's kindly ensure we follow the design correctly by updating your design to meet this missing part of the design below:

- Use the correct background color and add the missing background bur effects.
- Use the correct font weight for the cancel button.
- Align the links correctly to match the design. 


*Figma design*
Screenshot from 2023-04-17 17-55-00

*your design*
Screenshot from 2023-04-17 17-54-24

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,





Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!

## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Mobile menu works correctly 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style3.css (hunk 1 above)
Comment (AleWaweru) [reply style3.css]: done

**Review by emmiiorji — APPROVED** [review]
Hi @AleWaweru and @egichuhis, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>


## Good Highlights✔️

- Requested changes have been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**